src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,51 @@
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+
+interface IWeather_Event {
+  features?: (IFeaturesEntity)[] | null;
+}
+
+interface IFeaturesEntity {
+  properties: IProperties;
+}
+
+interface IProperties {
+  severity: string;
+  headline: string;
+  senderName: string;
+}
+
+export class WeatherEventsJob extends Job {
+  // Run once an hour
+  public interval: number = 1000 * 60 * 60;

```

Comment:
```suggestion
  public interval: number = moment.duration(1, 'hours').asMilliseconds();
```
More readable and does calculation for you

Reply:
Yeah I like that a lot better. If I apply it, though, I get an error saying "moment refers to a UMD global, but the current file is a module. Consider adding an import instead".

Reply:
oh I just realized I just need to import it.

Reply:
On second thought, I'm not sure if we want to import momentjs just to do this since moment is a very bloated library, but if we're using it somewhere else already then I have no issues with that.